### ISSUE TITLE
Deprecation: Document#tags is now a key in the #data hash.

### ISSUE THREAD

**Issue by johlym** [algolia/algoliasearch-jekyll#14]
I apologize if this is already mentioned somewhere (could not find it).

When I run `jekyll alogolia push` I get hundreds of these:

``` shell
        Deprecation: Document#tags is now a key in the #data hash.
                     Called by /usr/local/rvm/gems/ruby-2.2.3/gems/algoliasearch-jekyll-0.4.3/lib/record_extractor.rb:53:in `tags'.
        Deprecation: Document#slug is now a key in the #data hash.
                     Called by /usr/local/rvm/gems/ruby-2.2.3/gems/algoliasearch-jekyll-0.4.3/lib/record_extractor.rb:35:in `metadata'.
```

And it's a bit ugly during my deployment process. Any ideas on what's taking place or if there's anything I can do to mitigate this?

**Comment by pixelastic** [comment]
Thanks for the report @jelyman2. I'm actually already working on it. This stack of warning messages where introduced by Jekyll 3. If you stay with Jekyll 2.5, you should be ok.

I'm in the process of making the plugin compatible with both 2.5 and 3.0 but I haven't finished it yet.

**Comment by johlym** [comment]
Awesome, thanks for the update!

**Comment by pixelastic** [comment]
Just to let you know I did not forget about this issue, but the setup to run tests under two different Jekyll version is a bit tricky and I did not had time to complete it yet. But this bothers me as well, and this will be fixed soon :)

**Comment by borisschapira** [comment]
:+1:

**Comment by pixelastic** [comment]
This has been fixed with `v0.6.0` that I just released.

cc @borisschapira @jelyman2

**Comment by borisschapira** [comment]
:+1: Thanks !
